api/pkg/service/admin/service.go

```diff
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package auth
+package admin
```

Comment:
why we are moving to admin pkg?
it can be in auth pkg as a separate pkg ..
it is shared by catalog rating admin

Reply:
yes before it was in auth and used by admin, catalog and rating, so since we are removing auth hence just moved to admin. Should we create a separate package for it ??